src/coreclr/tools/Common/JitInterface/CorInfoTypes.cs

```diff
@@ -501,10 +501,6 @@ public enum CorInfoInlineTypeCheckSource
     public enum CorInfoInlineRestrictions
```

Comment:
None of these restrictions are used anymore. Delete the whole thing?

Reply:
Done. I've not done an actual JitInterface change to drop the unused parameter. I can do that to if needed at this point or it can wait for the next round of cleanups in JitInterface (e.g. when IL verification in RyuJIT is wiped).

Reply:
I would do it now.